lib/constructs/interface.js

```diff
@@ -172,6 +219,13 @@ Interface.prototype.generateExport = function () {
     this.setup(obj, constructorArgs, privateData);
     return utils.implForWrapper(obj);
   },
+  createDefaultIterator(target, kind) {
+    const iterator = Object.create(IteratorPrototype);
```

Comment:
Can we just `new` this up normally? I haven't checked the spec for how the object has to be structured.